metrics/mixin/Makefile

```diff
@@ -0,0 +1,29 @@
+# This Makefile should be used by developers to test generation of new new rules and alerts
```

Comment:
Let's link it to the main Makefile so we can run these from root.

Reply:
ack